src/wallet/test/wallet_test_fixture.cpp

```diff
@@ -0,0 +1,28 @@
+#include "wallet/test/wallet_test_fixture.h"
+
+#include "rpc/server.h"
+#include "wallet/db.h"
+#include "wallet/wallet.h"
+
+WalletTestingSetup::WalletTestingSetup(const std::string& chainName):
+    TestingSetup(chainName)
+{
+    bitdb.MakeMock();
+
+    bool fFirstRun;
+    pwalletMain = new CWallet("wallet.dat");
```

Comment:
Sounds good to me in principle (although I just moved this code here, I didn't touch it)